README.rst

```diff
@@ -108,37 +108,40 @@ Generate Terraform variables:
    seed_vm_flavor = "general.v1.small"
    seed_disk_size = 100
 
-   multinode_flavor     = "changeme"
-   multinode_image      = "CentOS-stream8-lvm"
+   multinode_flavor     = "general.v1.medium"
+   multinode_image      = "overcloud-rocky-9-yoga-20231026T081556"
```

Comment:
Is there any particular reason to use this image, and not ``Rocky9-lvm``?

Reply:
`Rocky9-lvm` gets updated _somewhat_ regularly. Pinning to a specific image improves reliability

Reply:
Plus making it more accurate to production is better

Reply:
Changed my mind, I've just remembered that the overcloud* images have some awkward issues with vault at the moment that haven't yet been resolved

Reply:
> Changed my mind, I've just remembered that the overcloud* images have some awkward issues with vault at the moment that haven't yet been resolved

Will want to change the Ubuntu image mentioned later in the document too

Reply:
Done